flowtools/metacyto_preprocess/metacyto_preprocess.xml

```diff
@@ -0,0 +1,103 @@
+<tool id="metacyto_preprocess" name="Pre-process samples" version="1.0">
+  <description>for MetaCyto</description>
+  <requirements>
+    <requirement type="package" version="1.42.0">bioconductor-flowcore</requirement>
```

Comment:
flowcore is a dependency of metacyto conda package (https://bioconda.github.io/recipes/bioconductor-metacyto/README.html#bioconductor-metacyto) so we can try removing the flowcore dependency here and test it to see if it is not pinning flowcore to 1.42 for a compatibility reason. If removing flowcore doesn't work, we will need to ask for a multi-tool container to be created [here](https://biocontainers.pro/#/multipackage) or [here](https://github.com/BioContainers/multi-package-containers) with both these packages.